java/src/main/native/OrtJniUtil.h

```diff
@@ -11,9 +11,12 @@
 extern "C" {
 #endif
 
-jsize safecast_size_t_to_jsize(size_t v);
-
-jsize safecast_int64_to_jsize(int64_t v);
+typedef struct {
+  size_t dimensions;
+  size_t arrSize;
```

Comment:
My understanding is that dimensions is the number of dims and arrSize is a byte size. Can we get by by having only one of them?

Reply:
I can document it, dimensions is the number of dimensions and `arrSize` is the number of elements in the tensor. It may change a little as I integrate the sparse tensor stuff.